<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="figure_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>950</width>
      <height>450</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>950</width>
      <height>450</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>120</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>效应类型</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QRadioButton" name="momentumButton">
        <property name="text">
         <string>动量效应</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="crossoversButton">
        <property name="text">
         <string>反转效应</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_index">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>455</y>
      <width>120</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>指数类型</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="index300Button">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>沪深300</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QRadioButton" name="index500Button">
        <property name="text">
         <string>中证500</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QRadioButton" name="index800Button">
        <property name="text">
         <string>中证800</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>计算收益率时间长度（月）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="periodSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>172</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>6</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_percent">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>455</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>组合占比</string>
    </property>
    <widget class="QCheckBox" name="checkBoxP5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5%</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxP10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>10%</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxP1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1%</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxP20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>20%</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxP30">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>30%</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textReturn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>460</y>
      <width>150</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_returntype">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>超额收益率类型</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>117</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QRadioButton" name="periodReturnButton">
        <property name="text">
         <string>分期超额收益率</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cumReturnButton">
        <property name="text">
         <string>累计超额收益率</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>groupBox_type</zorder>
   <zorder>figure_widget</zorder>
   <zorder>groupBox_index</zorder>
   <zorder>groupBox_percent</zorder>
   <zorder>textReturn</zorder>
   <zorder>groupBox_returntype</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
